src/EnergyPlus/HVACManager.cc

```diff
@@ -3024,7 +2872,7 @@ void SetHeatToReturnAirFlag(EnergyPlusData &state)
                     }
                 }
                 for (int spaceNum : thisZone.spaceIndexes) {
-                    auto &thisSpace = state.dataHeatBal->space(spaceNum);
+                    auto const &thisSpace = state.dataHeatBal->space(spaceNum);
```

Comment:
[src/EnergyPlus/HVACManager.cc:3027]:(style),[constVariable],Variable 'thisSpace' can be declared as reference to const